### ISSUE TITLE
Unable to add The Empire to Jackett

### ISSUE THREAD

**Issue by rjch-au** [Jackett/Jackett#1568]
When trying to add The Empire to Jackett, I get the message "Configuration failed: Login Failed, got redirected".  An enhanced log (with some information masked) shows that a login was successful.  The basic log was showing an entry "Error CardigannIndexer (theempire): Found captcha during automatic login, aborting".

[log.txt](https://github.com/Jackett/Jackett/files/1151041/log.txt)

Jackett version:  0.7.1533.0

**Comment by rjch-au** [comment]
Bump.  Still having this issue even after an upgrade to Jackett.  Is anyone successfully using The Empire with Jackett?

**Comment by flightlevel** [comment]
@adamwinn would you be able to have a look?

**Comment by adamwinn** [comment]
I could but I'm having issues with my local dev environment right now.  I go to add a tracker and it adds but the table of trackers on the main page is missing

**Comment by chibidev** [comment]
@adamwinn Upon testing #1584 I've encountered that the browser caches the JS file doing the ajax queries and tends to load from cache rather than from the server. Try to clear the cache to solve the problem. If it does not, look into the web console to see where the XHRs go and what's the response.

**Comment by rjch-au** [comment]
For what it's worth, the issue is still occurring with Jackett version 0.8.128.0.

**Comment by adamwinn** [comment]
I'm not sure what's going on here.  I didn't see `Found captcha during automatic login, aborting` but I also did get the `Login Failed, got redirected`.  Looking at the debug, it's saying I was logged in and even is showing the html for the page you get after you log in, so I'm definitely logged in.  I'm not sure why it's saying that error.

**Comment by kaso17** [comment]
`Login Failed, got redirected`
A request to main.php shouldn't result a redirect once you're logged in.

Otherwise you might want to try to change it to browse.php
```json
    test:
      path: main.php
```

**Comment by adamwinn** [comment]
You get redirected to main.php upon login.  That's their landing page.  So basically the `test` selector isn't correct?  So try changing that and see if it works?

**Comment by adamwinn** [comment]
It works obviously if I comment out the test.

```
    # test:
    #   path: /main.php    
```

**Comment by rjch-au** [comment]
It adds successfully, yes, but searches fail.


Sep 10, 2017 07:21:14 PM | Error | CardigannIndexer (theempire): Found captcha during automatic login, aborting
-- | -- | --
Sep 10, 2017 07:21:12 PM | Info | CardigannIndexer (theempire): Relogin required

**Comment by kaso17** [comment]
Looks like a deeper problem then (Login is successful but cookies are not retrieved correctly or something else is missing). I can't help much further without access.

**Comment by rjch-au** [comment]
If you need an invite, send me an email address and I'll send you one.

**Comment by adamwinn** [comment]
@rjch-au I believe @kaso17 is at kaso1717 -at- gmail.com.  That's what's listed in the readme

**Comment by rjch-au** [comment]
@kaso17 Invite sent.  Let me know if there are any problems.

**Comment by kaso17** [comment]
thank you for the invite.
They're enforcing https now.
Just change the site link to `https://theempire.click`
Fixed via e33e95c1b284237647c34827f6137dac838e4d4e

**Comment by rjch-au** [comment]
Fix confirmed.  The Empire is now adding and searching fine in Jackett.  Thanks for the help.
